tf2onnx/custom_opsets/onnx_ml.py

```diff
@@ -42,10 +42,17 @@ def version_8(cls, ctx, node, initialized_tables, **kwargs):
 
         dtype = ctx.get_dtype(node.output[0])
         in_dtype = ctx.get_dtype(node.input[1])
-        utils.make_sure(dtype == TensorProto.INT64 and in_dtype == TensorProto.STRING,
-                        "Only lookup tables of type string->int64 are currently supported.")
+        utils.make_sure((dtype == TensorProto.INT64 and in_dtype == TensorProto.STRING) or
+                        (dtype == TensorProto.STRING and in_dtype == TensorProto.INT64),
+                        f"Only lookup tables of type string<->int64 are currently supported.")
+
+        if in_dtype == TensorProto.STRING:
+            cats_strings, cats_int64s = initialized_tables[shared_name]
+            attr = {'cats_int64s': cats_int64s, 'cats_strings': cats_strings, 'default_int64': default_val}
```

Comment:
This could be moved out of this if-block.

Reply:
done